### PR TITLE
Apparmor as alternative to SELinux in auditd

### PR DESCRIPTION
**Pull request by micheelengronne**

### CODE REVIEW

**Comment by micheelengronne** [comment]
I have read the Mondoo CLA Document and I hereby sign the CLA

**Review by atomic111 — APPROVED** [review]
@micheelengronne thanks for the nice addition, great!!!

File: core/mondoo-linux-security.mql.yaml [hunk 1]
```diff
@@ -1430,14 +1430,22 @@ queries:
...
-        Add the following lines:
+        Add the following lines, for SELinux:
 
         ```
         -w /etc/selinux/ -p wa -k MAC-policy
 
         -w /usr/share/selinux/ -p wa -k MAC-policy
         ```
 
+        Add the following lines, for Apparmor:
```
Comment (tas50) [inline core/mondoo-linux-security.mql.yaml:1441]:
```suggestion
        Add the following lines, for AppArmor:
```

**Comment by tas50** [comment]
Thanks @micheelengronne!

**Pull request merged**